script.transmission/README.md

```diff
@@ -0,0 +1,26 @@
+Transmission-XBMC
```

Comment:
please change to Transmission for Kodi (this naming is not allowed as it violates our trademark)